ui/lib/client/index.tsx

```diff
@@ -59,7 +59,9 @@ function initAxios() {
       }
     }
     const content = i18next.t(errCode)
-    err.message = content
+    // err.message = content // sometimes we can't change err.message at all, still don't know the exact reason
```

Comment:
I encourage to dig deeper about this behaviour.

Reply:
Ok!